### ISSUE TITLE
Posting empty declaration does not clear DO, rollback for some attributes does not work

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-declarative-onboarding#118]
### Environment
 * Declarative Onboarding Version: 1.10.0-2
 * BIG-IP Version: 14.1.2.0.0.37

### Summary
Issuing an empty declaration to a failed (rolledback) declaration does not properly clear DO state.

### Steps To Reproduce

Produce a rolled back (erred) declaration for DO with Dbvariables class members set.

1. Submit the following declaration:
```json
{
    "schemaVersion": "1.6.0",
    "class": "Device",
}
```

2. Observe the following error response:
```json
{
    "id": "ca8f4391-4590-405a-9083-c8dbc771eae4",
    "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/ca8f4391-4590-405a-9083-c8dbc771eae4",
    "code": 500,
    "status": "ERROR",
    "message": "invalid config - rolled back",
    "errors": [
        "Cannot read property 'DbVariables' of undefined"
    ],
    "result": {
        "class": "Result",
        "code": 500,
        "status": "ERROR",
        "message": "invalid config - rolled back",
        "errors": [
            "Cannot read property 'DbVariables' of undefined"
        ]
    },
    "declaration": {
        "async": true,
        "schemaVersion": "1.0.0",
        "class": "Device"
    }
}
```

### Expected Behavior
Clear DO.

### Actual Behavior
500 errors

**Comment by dstokesf5** [comment]
It looks like an empty DO declaration should include an empty tenant:
```json
{
    "schemaVersion": "1.6.0",
    "class": "Device",
    "Common": {
    	"class": "Tenant"
    }
}
```
We will need to clean up the case where the tenant is left off. Either by having a default tenant, or having the schema flag a declaration without a tenant as invalid.

**Comment by dstokesf5** [comment]
Added AUTOTOOL-1238 to our internal product backlog to improve this.

**Comment by dstokesf5** [comment]
This issue has been resolved in the 1.19.0 release.
